src/bank_forks.rs

```diff
@@ -18,11 +18,28 @@ impl BankForks {
             banks,
         }
     }
-
+    pub fn frozen_banks(&self) -> Vec<u64> {
+       self.banks
+            .iter()
+            .filter(|(k, v)| v.is_frozen())
+            .map(|x| *x.0)
+            .collect()
+    }
+    pub fn live_banks(&self) -> Vec<u64> {
```

Comment:
the banks in BankForks should all be live...  blocktree_processor populates BankForks with children, and insert() removes parents...

Reply:
This is where we were using the term `active` in the proposal, not `live`.